### DISCUSSION TITLE
Should a user be able to archive an already archived folder?

### DISCUSSION THREAD

**Discussion by macheitor** [dirkpetersen/froster#29] [General]
What if a user tries to archive an already archived folder? I'm thinking of strange use cases where the user adds a file to an already archived folder. This folder may have been cleaned/deleted or not. Also, the entry in 'froster-archives.json' will be overwritten...

I think we should take the safest path and avoid allowing the user to archive an already archived folder. There is a message that suggests the user rename the '.froster.md5sum' file to proceed. This is fine when an error occurred while uploading files because it uploads everything again, but wrong if the user has already archived a folder and is 'playing around' by adding additional files with or without deleting the folders.

I will forbid the user from taking any action if the '.froster.md5sum' file exists, and I will add a flag that allows the user to 'play around', something like 'froster archive --add-files /already/archived/folder/path'.

However, this feature will be developed once the basic Froster features are completed and tested.

**Comment by dirkpetersen** [comment]
I agree, sound like a plan. And let's also move this messaging into the troubleshooting section of the documentation where this could be explained with more nuance:, and the error message could be "Sorry, this folder has already been archived, if you would like to ....... please check the troubleshooting section at http://...... for options
